cmd/kubermatic-api/options.go

```diff
@@ -41,7 +41,6 @@ type serverRunOptions struct {
 	internalAddr   string
 	prometheusURL  string
 	workerName     string
-	presetsFile    string
```

Comment:
You should also remove `-dynamic-presets`, which would always be true anyway. Removing the field from `serverRunOptions` will then trickle down into the other parts of the code where you can remove more code.

Reply:
Done. Getting rid of dynamicPresets simplified a bit preset's provider logic.